src/modules/writeLog.js

```diff
@@ -0,0 +1,70 @@
+const fs = require('fs');
+
+const childProcess = require('child_process');
+const os = require('os');
+
+const logFileName = './src/logs/activityMonitor.log';
+const timer = (ms) => new Promise((res) => setTimeout(res, ms));
+
+function execProcess(command) {
+    let result = [];
+
+    childProcess.exec(command, { timeout: 5 * 1000 }, (err, stdout) => {
+        if (err) {
+            console.log(`error: ${err}`);
+            return;
+        }
+
+        const lines = stdout.toString().split('\r\n');
+
+        lines.forEach(function (line) {
+            let lineItem = line;
+            result.push(lineItem);
+        });
+    });
+
+    return result;
+}
+
+function programUses() {
+    const currOS = os.platform();
+    let result = [];
```

Comment:
I'd suggest using object to store commands, could go like so:
```
const COMMANDS = {
  win32: 'powershell ...',
  darwin: 'ps ...',
  linux: 'ps ...'
}
...
execProcess(COMMANDS[currOS])
// could do fallback to 'ls' command, or throw in case no value in COMMANDS, but approach with 'result' is also ok
```